hpOneView/resources/settings/scopes.py

```diff
@@ -131,8 +92,7 @@ def update(self, resource, timeout=-1):
 
         """
         headers = {'If-Match': resource.get('eTag', '*')}
-        return self._client.update(resource, timeout=timeout, default_values=self.DEFAULT_VALUES,
-                                   custom_headers=headers)
+        return super(Scopes, self).update(resource, timeout=timeout, default_values=self.DEFAULT_VALUES, custom_headers=headers)
 
```

Comment:
update() method in Resource class don't accept 'default_values' as argument. So don't pass that argument. init() method in Resource class will anyway fetch the default values.

Reply:
I have updated scopes and test_scopes files according the change you mentioned here.